templates/account/login.html

```diff
@@ -9,7 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{{ STATIC_URL }}images/favicon.png">
 
-    <title>{% trans  "Signin on Libreosto" %}</title>
+    <title>{% trans  "Signin on Libreosteo" %}</title>
```

Comment:
I think that require a change in translations file

Reply:
Yes you're right, i try tout execute tests before 😊

Reply:
fixed on the branch